chunky/src/java/se/llbit/chunky/ui/controller/ChunkyFxController.java

```diff
@@ -193,7 +193,9 @@ public void showRenderPreview() {
   static class GUIRenderListener implements RenderStatusListener {
     private final ChunkyFxController gui;
     private int spp;
-    private int sps;
+    private LinkedList<Integer> sps_queue = new LinkedList<Integer>();
+    private static final int AVERAGE_PERIOD = 150;
```

Comment:
A `LinkedList` has a lot of overhead. We could simply use a `int[AVERAGE_PERIOD]` instead.

Reply:
Oh yeah - That was left over from when I was calculating the average slightly differently